### DISCUSSION TITLE
Is this mod available on MAC OS?

### DISCUSSION THREAD

**Discussion by SomeEthing** [maximmax42/Discord-CustomRP#216] [Q&A]
Is this mod available on MAC OS or only for windows ?

**Comment by maximmax42** [comment]
This isn't a mod, but no, Windows only.
You can either use Vencord with CustomRPC plugin (not made by me) or use https://drpcm.t1c.dev (also not made by me).

**Reply by SomeEthing** [reply]
Ah ok, thanks dude
